pages/common/git-browse.md

```diff
@@ -0,0 +1,9 @@
+# git browse
+
+> View a upstream repository within the browser.
```

Comment:
```suggestion
> View an upstream repository in the default web browser.
```

Reply:
We use `default browser` is most pages. I think we should unify that.

Reply:
Can't browser also be used in other context? To me it sounds like `browser` is only used colloquially.

Reply:
The verb `browse` could be used in another context, but can you find another usage for `browser`?

Reply:
A file browser first comes to mind.

Reply:
`Nautilus` and `dolphin` present itself as `file manager`, sometimes we called them `file explorer`s but not `browser`s.

Reply:
Unifying it to `browser` sounds good. I like `web browser`, but that seems to be a personal opinion, rather than a general one.